packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -208,17 +205,14 @@ async function fetchLatestSnapshotCore(
             async (event) => {
                 const startTime = performance.now();
                 const response = await snapshotDownloader(
-                    url,
-                    {
-                        body: postBody,
-                        headers,
-                        signal: controller?.signal,
-                        method: "POST",
-                    },
-                ) as IOdspResponse<IOdspSnapshot>;
+                    odspResolvedUrl,
+                    storageToken,
+                    snapshotOptions,
+                    controller,
+                );
                 const endTime = performance.now();
                 const overallTime = endTime - startTime;
-                const snapshot = convertOdspSnapshotToSnapsohtTreeAndBlobs(response.content);
+                const snapshot = response.response.content;
```

Comment:
response.response suggests that we should rename variable to something more representing? Or rename property? Something that makes it more readable